src/engraving/playback/playbackcontext.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef MU_ENGRAVING_PLAYBACKCONTEXT_H
+#define MU_ENGRAVING_PLAYBACKCONTEXT_H
+
+#include "mpe/events.h"
+
+#include "libmscore/note.h"
+
+#include "playback/utils/arrangementutils.h"
+#include "playback/utils/pitchutils.h"
+
+namespace mu::engraving {
+struct PlaybackContext {
+    mpe::timestamp_t nominalTimestamp = 0;
+    mpe::duration_t nominalDuration = 0;
+    mpe::dynamic_level_t nominalDynamicLevel = 0;
+    int nominalPositionTick = 0;
+    int nominalDurationTicks = 0;
+    qreal beatsPerSecond = 0;
```

Comment:
I recently added a separate type for this :) https://github.com/musescore/MuseScore/blob/master/src/engraving/types/bps.h

Reply:
Heh, I actually noticed this type when resolved conflicts, but forgot to change this particular one

I'll refine it in the next PR